test/PluginExample.hs

```diff
@@ -16,9 +16,9 @@ main = pure ()
 #else
 
 {-# OPTIONS_GHC -fplugin=RecordDotPreprocessor #-}
-{-# LANGUAGE DuplicateRecordFields, TypeApplications, FlexibleContexts, DataKinds, MultiParamTypeClasses, TypeSynonymInstances, FlexibleInstances #-}
+{-# LANGUAGE DuplicateRecordFields, TypeApplications, FlexibleContexts, DataKinds, MultiParamTypeClasses, TypeSynonymInstances, FlexibleInstances, TypeFamilies, TypeOperators, GADTs, UndecidableInstances #-}
 -- things that are now treated as comments
-{-# OPTIONS_GHC -Werror -Wall -Wno-type-defaults -Wno-partial-type-signatures -Wincomplete-record-updates -Wno-unused-top-binds #-}
+{-# OPTIONS_GHC -Wall -Wno-type-defaults -Wno-partial-type-signatures -Wincomplete-record-updates -Wno-unused-top-binds #-}
```

Comment:
This has ended up as a conflict. Can you rebase over master? Why remove the `-Werror`?

Reply:
Well, I had these problems with werror. I'll try to fix if it possible

```
record-dot-preprocessor/PluginExample:1:1: error: [-Wincomplete-record-updates, -Werror=incomplete-record-updates]
    Pattern match(es) are non-exhaustive
    In a record-update construct: Patterns not matched: (Foo8 _ _)
            
record-dot-preprocessor/PluginExample:1:1: error: [-Wincomplete-record-updates, -Werror=incomplete-record-updates]
    Pattern match(es) are non-exhaustive
    In a record-update construct: Patterns not matched: (Quux8 _)
            
record-dot-preprocessor/PluginExample:1:1: error: [-Wincomplete-record-updates, -Werror=incomplete-record-updates]
    Pattern match(es) are non-exhaustive
    In a record-update construct: Patterns not matched: (Quux8 _)
            
record-dot-preprocessor/PluginExample:1:1: error: [-Wincomplete-record-updates, -Werror=incomplete-record-updates]
    Pattern match(es) are non-exhaustive
    In a record-update construct: Patterns not matched: (Nonhuman _)
```

Reply:
Oh, it works after pulling from upstream master